README.md

```diff
@@ -8,6 +8,7 @@ const method = new ValidatedMethod({
   name, // DDP method name
   mixins, // Method extensions
   validate, // argument validation
+  callOptions, // options passed to Meteor.apply
```

Comment:
Yeah, I pondered on that for a bit.  I went with callOptions since the way you end up calling the method is `methodName.call(...)`.  I thought callOptions would make more sense, but then when I tried to explain what it did, I kinda had to say `meteor.apply`.  Sigh.  What would you prefer?